tests/framework/config/cluster.go

```diff
@@ -28,3 +28,8 @@ type ClusterConfig struct {
 	ClientTLS         TLSConfig
 	QuotaBackendBytes int64
 }
+
+type NamedClusterConfig struct {
```

Comment:
Don't think this is needed if we use anonymous function in `TestTxnSucc` and `TestTxnFail` as proposed in comment above.

Reply:
updated